### PR TITLE
V2: Views: Buttons & Pills

### PR DESCRIPTION
**Pull request by gblakeman**
A buildout of the various icons, buttons, and pills we will need for the `v2` UI as individual components.

realish-ui

### CODE REVIEW

File: src/views/App.svelte [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  import ItemsList from './ItemsList';
+  import ButtonsList from './style-guide/ButtonsList';
+  import FontPreload from './FontPreload';
   import FooterBar from './FooterBar';
+  import HeaderBar from './HeaderBar';
+  import IconsList from './style-guide/IconsList';
+  import ImageSandbox from './ImageSandbox';
+  import PillsList from './style-guide/PillsList';
+  import TypeList from './style-guide/TypeList';
+  // import ItemsList from './ItemsList';
```
Comment (joeyyang) [inline src/views/App.svelte:14]: dangling comment import - don't mind leaving it in if you're using it later but wanted to make sure you saw

File: src/views/forms-controls/PillActionPrimary.svelte [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  export let isIconLeft = false;
+  export let label = 'Button';
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<style lang="scss">
+  @import "./src/assets/css/base";
+
+  button {
```
Comment (joeyyang) [inline src/views/forms-controls/PillActionPrimary.svelte:21]: noticed a lot of these `button` styles are redundant - does it make sense to create a base `button` class that we extend in each individual `.svelte` file so it's clear what styles are specific to each button?

**Review by lynnerang — APPROVED** [review]
Looks great, thanks for sharing!

**Review by joeyyang — APPROVED** [review]
One suggestion about cleaning up some of the styles in individual component files, but otherwise looks good!

File: src/views/forms-controls/PillActionPrimary.svelte (hunk 2 above)
Comment (gblakeman) [reply src/views/forms-controls/PillActionPrimary.svelte:21]: I’ve gone back and forth on this. My current thinking is that even though there is redundancy, with the current setup we can set the expectation that _each component contains all of its styles independently_, without any conflicts with other components.

If `PillActionPrimary` drifts visually over time from `ButtonActionSecondary` the developer making edits at the time only has to worry about the CSS in the component they’re directly editing without any cascading effects. Ultimately, that seemed like the most advantageous approach to me, thinking longer term, despite the redundant code today.

Thoughts on that framing?

/cc @lynnerang

File: src/views/App.svelte (hunk 1 above)
Comment (gblakeman) [reply src/views/App.svelte:14]: Yes, this will get used again, but I appreciate the check!

**Pull request merged**